src/Module/Hashtag.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @file src/Module/Hashtag.php
+ */
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Core\System;
+use dba;
+
+/**
+ * Hashtag module.
+ */
+class Hashtag extends BaseModule
+{
+
+	public static function content()
+	{
+		$result = [];
+
+		$t = escape_tags($_REQUEST['t']);
```

Comment:
Please add a `require_once 'include/text.php';` call, it will be easier to move it to `src/` later.